taxi/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.contrib import admin
+from django.urls import path, include
```

Comment:
The `include` function is imported but not used in this file. Consider removing it to clean up the code.